types/hooks/screens/useChatScreenHook.types.ts

```diff
@@ -5,7 +5,7 @@ interface UseChatScreenHook {
   chats: ChatSchema[];
   goBackFromChatScreen: () => void;
   goToChatInfoScreen: (params?: any) => void;
-  sendChat: (message: string) => Promise<void>;
+  sendChat: (message: string, attachments: any) => Promise<void>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function now includes an `attachments` parameter. However, the type of `attachments` is defined as `any`, which is not a good practice in TypeScript. It's better to define a specific type for `attachments` to take advantage of TypeScript's static typing. If `attachments` is an array of objects, you could define an interface for the attachment object and then use that interface in the array type.

```diff
- sendChat: (message: string, attachments: any) => Promise<void>;
+ sendChat: (message: string, attachments: AttachmentType[]) => Promise<void>;
```
In this case, `AttachmentType` would be an interface that you define to describe the shape of an attachment object. This will make your code more robust and less prone to runtime errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->